packages/console/src/pages/ConnectorDetails/index.tsx

```diff
@@ -172,7 +170,6 @@ const ConnectorDetails = () => {
             </TabNavItem>
           </TabNav>
           <ConnectorContent
-            isDeleted={isDeleted}
```

Comment:
We should not remove this `isDeleted` logic; this causes a bug when deleting the connector, which has unsaved changes.

Reply:
Wasn't able to reproduce it last night. Maybe I forgot the steps.

Reply:
You are right. Code updated. @xiaoyijun